### PR TITLE
[GHO-61] Fix typography

### PR DESCRIPTION
**Pull request by orakili**
Ticket: GHO-61

This is a follow-up of #103 and a replacement of #118 with typography fixes.

Both in #103 and in this PR, the typography values come from the `Gho-VisualAdjusments.pdf` (most recent document) with missing information taken from the `Gho-CheatSheet.pdf` (second most recent doc) or the `Gho-CSS.rtf` document (oldest document among the 3).

In addition to the typography changes, I changed the color of the `arrows` for the **further reading** links and the **related articles** to the UN blue to match the arrows from the main menu because the colors that were used didn't match anything mentioned in the above documents. 

We can revisit the arrow color when tackling GHO-69 that deals with the color of the links.

### CODE REVIEW

File: html/themes/custom/common_design_subtheme/components/gho-related-articles/gho-related-articles.css [hunk 1]
```diff
@@ -79,7 +83,8 @@
 .gho-related-article__title {
   font-size: 1.125em;
   font-weight: 700;
-  margin: 1rem 0;
+  line-height: 2rem;
+  margin: 1rem 0 0.625rem 0;
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/components/gho-related-articles/gho-related-articles.css:87]: The change to the bottom margin is to match the `para spacing: 10pt` indication from the CheatSheet:

Screen Shot 2020-11-19 at 15 37 56

File: html/themes/custom/common_design_subtheme/components/gho-related-articles/gho-related-articles.css [hunk 2]
```diff
@@ -14,14 +14,18 @@
...
+  line-height: 2rem;
 }
 
 /**
  * Individual Article
  */
 .gho-related-article {
   margin-top: 2rem;
+  font-size: 1rem;
+  line-height: 1.75rem;
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/components/gho-related-articles/gho-related-articles.css:28]: That comes from the CheatSheet `font-size: 16pt, line-height: 28pt`:

Screen Shot 2020-11-19 at 15 51 58

File: html/themes/custom/common_design_subtheme/components/gho-text/gho-text.css [hunk 3]
```diff
@@ -23,10 +23,11 @@
     line-height: 2.25rem;
   }
 }
+/* Same styling as caption. @see gho-caption */
 .gho-text blockquote {
   margin: 2rem 0;
-  font-size: 0.875rem;
-  line-height: 1.5rem;
+  font-size: 0.75rem;
+  line-height: 1.25rem;
```
Comment (rupl) [inline html/themes/custom/common_design_subtheme/components/gho-text/gho-text.css:30]: This was subject to merge conflicts but I _think_ I got it right? The diff is quite small so I feel like I eliminated the correct pieces.

Comment (orakili) [reply html/themes/custom/common_design_subtheme/components/gho-text/gho-text.css:30]: @rupl That's the opposite. The file should have

```css
/* Same styling as caption. @see gho-caption */
.paragraph--type--text blockquote {
  margin: 2rem 0;
  font-size: 0.75rem;
  line-height: 1.25rem;
  color: #707070;
}
@media (min-width: 768px) {
  .paragraph--type--text blockquote {
    font-size: 0.875rem;
    line-height: 1.5rem;
  }
}
```

Comment (orakili) [reply html/themes/custom/common_design_subtheme/components/gho-text/gho-text.css:30]: I've pushed a commit to restore the media query with also the class name change.

Comment (rupl) [reply html/themes/custom/common_design_subtheme/components/gho-text/gho-text.css:30]: Thanks!

**Review by rupl — APPROVED** [review]
Provided that he meant `px` when he wrote `pt` in the Gho-VisualAdjustments.pdf file, all of this is correct. If he truly wanted `pt` then we'll need to multiply everything by 1.5 😂

**Pull request merged**